src/styles/units/_mixins.scss

```diff
@@ -0,0 +1,10 @@
+@mixin makeCircle($size) {
+  width: $size;
+  height: $size;
+}
+
+@mixin positionAbsolute($watch-size, $el-size) {
+  position: absolute;
+  left: ($watch-size - $el-size) / 2;
+  bottom: ($watch-size - $el-size) / 2;
+}
```

Comment:
It looks like "position-centered". And is not flexible to be just `positionAbsolute`
In this case you can just use
```
  position: absolute;
  left: 50%;
  bottom: 50%;
  tranform: translate(-50%,-50%)
```

Reply:
changed